app/models/user.js

```diff
@@ -46,5 +46,11 @@ module.exports = (sequelize, DataTypes) => {
     return User.findOne({ where: { email: e } });
   };
 
+  User.getUsers = (page, size) => {
+    if (page < 1) throw errors.parametersInvalid;
+    const offst = size * page - size;
+    return User.findAll({ offset: offst, limit: size });
```

Comment:
If you call the variables equal to the attributes you can use a shortcut and write the object like `{ object, limit }` and it will work.